tests/roles/mariadb_copy/tasks/main.yaml

```diff
@@ -57,6 +57,15 @@
 
     EOF
 
+- name: Modify Neutron db name
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
```

Comment:
do you need all the headers for a simple `sed` call? You don't call `oc` or `mariadb-client` here.

Reply:
also, I see above that the other task that creates the file does `cd {{ mariadb_copy_tmp_dir }}`. Should you do the same here to make sure you are in the right directory?

Reply:
I don't need any of the headers. I thought it was common practice to add them to any task.
And good eyes, I'm on wrong directory, will fix on next patch! :)

Reply:
It's not common practice for *any* script. These template substitutions will initialize some variables that are needed for `oc` and `mysql` client invocation. Since you don't use any of it, these are a waste.